frontend/src/components/pages/AdminDashboard/ViewDonationsAndCheckins.tsx

```diff
@@ -138,12 +166,13 @@ const ViewDonations = ({
         <Calendar
           key={selectedDay?.toString()}
           selectedDay={selectedDay as Date}
-          schedules={schedules}
+          schedules={isCheckInView ? checkIns : schedules}

```

Comment:
Can we rename it to smth other than schedules, it can get confusing if they can also be checkins (maybe schedulesOrCheckins lol)